roles/syslog-client/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,183 @@
+###################### Filebeat Configuration Example #########################
```

Comment:
Is there a way to tag the messages from test and pilot at the source, so that we can easily separate them in Elasticsearch?

Reply:
Yes, if we have test/pilot specific strings in variables?

Reply:
I've added a `environment_name` ("test", "pilot", "vm") to the groups_vars